apps/src/lib/kits/maker/util/browserChecks.js

```diff
@@ -1,4 +1,7 @@
 /** @file Some misc. browser check methods for maker */
+/* global SerialPort */ // Maybe provided by the Code.org Browser
```

Comment:
Are we not sure?

Reply:
Oh I've never read it like that! We are sure that if we're in the Code.org (Maker) Browser, then we have the SerialPort. I believe the 'Maybe' applies to that we might be in the Code.org Browser or in the Chrome Browser with the Chrome App (Chromebook).